processccd.rst

```diff
@@ -18,11 +18,11 @@ catalogs (``background`` and ``sourceCat`` fields of a
 In more detail, ProcessCcdTask executes the following steps:
 
 
-- 1.  ``Instrument Signature Removal`` -- Implemented by the :doc:`IsrTask <isrtask>` sub-task (isr configuration), this step removes CCD signatures (such as bias, dark current, flat-fielding, and cross-talk) and masks bad pixels.
+1.  ``Instrument Signature Removal`` -- Implemented by the :doc:`IsrTask <isrtask>` sub-task (isr configuration), this step removes CCD signatures (such as bias, dark current, flat-fielding, and cross-talk) and masks bad pixels.
```

Comment:
Took out the unordered list directives (the '--') and just left the ordered list numbering.